scripts/run_tests_in_ci.js

```diff
@@ -21,6 +21,9 @@ const { spawn } = require('child-process-promise');
 const { writeFileSync } = require('fs');
 
 const LOGDIR = process.env.CI ? process.env.HOME : '/tmp';
+// Maps the packages that have standalone cross browser testing to the necessary test script.
```

Comment:
It happens to be that we need these for Firestore and Auth, and also that they happen to not be part of test:changed core, but that's more of a coincidence. The distinguishing feature is that these are packages where we should not run `test:all` (or whatever argument is passed to the script using the `-s` flag in the package's version of `test:ci`) for cross-browser tests. Storage basically has the same problem but we solved it by editing karma.conf.js instead. We may actually want to change storage (in the future, in another PR) to use this same method, for consistency.